packages/ckeditor5-fullscreen/src/fullscreencommand.ts

```diff
@@ -78,8 +78,22 @@ export default class FullscreenCommand extends Command {
 	 * Disables the fullscreen mode.
 	 */
 	private _disableFullscreenMode(): void {
-		this._fullscreenHandler.disable();
+		this.fullscreenHandler.disable();
 
 		this.value = false;
 	}
 }
+
+/**
+ * Classic editor typeguard.
+ */
+function isClassicEditor( editor: ClassicEditor | DecoupledEditor | Editor ): editor is ClassicEditor {
```

Comment:
Why these extra types (`ClassicEditor | DecoupledEditor`) are needed for the parameter? Isn't `Editor` just enough?

Also, does TS really requires you to specify the type for `editor.constructor`? It doesn't seem like there's any ambiguity here, it should get resolved 🤔.